modules/logging-loki-cli-install.adoc

```diff
@@ -0,0 +1,197 @@
+// Module is included in the following assemblies:
+//
+:_content-type: PROCEDURE
+[id="logging-loki-cli-install_{context}"]
+= Installing logging Operators using the {product-title} CLI
+
+To install and configure logging on your {product-title} cluster, additional Operators must be installed. This can be done from the {product-title} CLI.
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+.Prerequisites
+
+* Supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+--
+include::snippets/logging-create-namespace-cli-snip.adoc[leveloffset=+1]
+--
+
+[NOTE]
+====
+If the `openshift-logging` namespace already exists, you must add the namespace label, `openshift.io/cluster-monitoring: "true"`, to enable metrics service discovery.
+====
+
+. Install the `Loki Operator` by creating the following objects:
+
+.. Create an Operator Group object YAML file (for example, `loki-og.yaml`) for the Red Hat OpenShift Logging Operator:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: logging-loki
+  namespace: openshift-operators-redhat <1>
+spec:
```

Comment:
Loki-operator should be deployed as a global operators.   We needn't specify target namespace here.

Reply:
```
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: logging-loki
  namespace: openshift-operators-redhat
  spec: {}
```